scripts/jenkins/master.sh

```diff
@@ -17,14 +17,7 @@ function init {
 
 function build {
   cd ../..
-
-  if [[ $NO_TEST == "true" ]]; then
```

Comment:
Added at 4bcb307abc29d35fa1fa3a6fd3ab1ef39dd52576 (and similar commits in each of the other branches).